api_v3/lib/types/resource/KalturaEntryResource.php

```diff
@@ -51,15 +51,12 @@ public function validateForUsage($sourceObject, $propertiesToSkip = array())
 		
 		$key = $srcFlavorAsset->getSyncKey(asset::FILE_SYNC_ASSET_SUB_TYPE_ASSET);
 		$c = FileSyncPeer::getCriteriaForFileSyncKey($key);
-		$c->addAnd(FileSyncPeer::FILE_TYPE, array(FileSync::FILE_SYNC_FILE_TYPE_FILE, FileSync::FILE_SYNC_FILE_TYPE_LINK), Criteria::IN);
+		$c->addAnd(FileSyncPeer::FILE_TYPE, array(FileSync::FILE_SYNC_FILE_TYPE_FILE, FileSync::FILE_SYNC_FILE_TYPE_URL), Criteria::IN);
```

Comment:
Why did you remove links?
I think you should remove this condition all together and enable links.